test/controllers/graphql_controller_12_test.rb

```diff
@@ -908,25 +908,93 @@ def teardown
     assert_equal 'This item already exists', JSON.parse(@response.body).dig('errors', 0, 'message')
   end
 
-  test "should get annotations count" do
```

Comment:
Request: I don't think that removing this test was intended? I noticed that this PR uncovers line 19 of `app/graph/types/inclusions/task_and_annotation_fields.rb`.